python/lsst/consdb/pqserver.py

```diff
@@ -529,9 +567,7 @@ def get_flexible_metadata_keys(
     _ = instrument_tables.compute_flexible_metadata_table_name(instrument, obs_type)
     instrument_tables.refresh_flexible_metadata_schema(instrument, obs_type)
 
-    return FlexMetadataSchemaResponseModel(
-        schema=instrument_tables.flexible_metadata_schemas[instrument.lower()][obs_type]
-    )
+    return instrument_tables.flexible_metadata_schemas[instrument][obs_type]
```

Comment:
I take it `.lower()` isn't needed anymore?

Reply:
It is needed but it happens on line 565.